test/ValueIdTest.cpp

```diff
@@ -459,3 +459,12 @@ TEST_F(ValueIdTest, EncodedIriEqualityWithLocalVocabEntry) {
     EXPECT_LT(localVocabId2, encodedId) << inconsistentOrderingMessage;
   }
 }
+
+// Note:: the `isTrivial` functionality is also tested using `static_assert`s
+// across the codebase, hence we don't test it exhaustively here, but only
+// please the coverage tool.
+TEST(ValueId, isTrivial) {
+  EXPECT_TRUE(Id::makeUndefined().isTrivial());
+  EXPECT_FALSE(
+      Id::makeFromBlankNodeIndex(BlankNodeIndex::make(17)).isTrivial());
```

Comment:
The other edge case "EncodedVal" would also be nice to see here, but other than that this looks good.